### PR TITLE
Imported the 'our refreshed vision' doc

### PR DESCRIPTION
**Pull request by peteryates**

### CODE REVIEW

File: app/posts/cross-programme/our-refreshed-vision.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+2. We have maximised value for users and increased efficiency through automation
+3. We have stabilised the services and consolidated down
+
+## Phase 1 -- Our services serve their core purposes and are flexible enough to adapt to future changes and opportunities
+
+Our services still fall short of delivering their core functions and are incredibly rigid, making it difficult to respond to new scenarios and changes. The lack of flexibility means that any future big changes to the delivery model will require a significant amount of work -- potentially building again from scratch rather than building from what we already have.
+
+Our key goals in this phase are to:
+
+- Decouple the NPQ service from the ECF service so they are separate services and databases to eliminate unnecessary dependencies and simplify them. This should eliminate a lot of the problems we have around data integrity.
```
Comment (claire-hughez) [inline app/posts/cross-programme/our-refreshed-vision.md:85]: technically each bullet point should not start with a capital letter and not be more than 1 sentence (without a full stop...)

File: app/posts/cross-programme/our-refreshed-vision.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Phase 2 - We have maximised value for users and increased efficiency through automation
+
+Once we have got the fundamentals in place and have the foundations for
+a more sustainable and flexible service in place, we can prioritise
+efficiency gains and optimising the services. This includes:
+
+- Reshaping the programme to give the NPQ reg team ownership of the end-to-end journey from registration through to the API and payments. This will lead to a more efficient use of resource and a more empowered team owning NPQ delivery.
+- Increasing automation and self-serve to achieve greater efficiency and lower costs for programme delivery (for DfE and providers). -   Increasing the scope of the services to deliver end-to-end, by including NPQ completion data and induction data collection to minimise data entry duplication for our users and confusion.
+- Better meeting user needs around knowing what DfE CPD options there are at different stages in their career. ## Phase 3 - We have stabilised the services and consolidated down
```
Comment (claire-hughez) [inline app/posts/cross-programme/our-refreshed-vision.md:98]:
```suggestion
- Better meeting user needs around knowing what DfE CPD options there are at different stages in their career. 

## Phase 3 - We have stabilised the services and consolidated down
```

File: app/posts/cross-programme/our-refreshed-vision.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+By the time we get to phase 3, we will be able to reduce the number of people needed to run the services. The services should be set up to easily accommodate funding and minor policy changes with minimal digital resource required.
+
+This frees people up to go and tackle other problems along the teacher service line! Separating the NPQ service from the ECF application and the different levels of complexity associated with both policies and services means that the NPQ and ECF services will progress through the phases at different rates.
```
Comment (claire-hughez) [inline app/posts/cross-programme/our-refreshed-vision.md:102]: 'along the teacher service line' is weird... what is meant by this?

File: app/posts/cross-programme/our-refreshed-vision.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+![Phases for ECF and NPQ for 2025 and 2026](/cross-programme/our-refreshed-vision/phases.png)
+
+This is our vision and our strategy for getting there, but it's not static. We are continuously updating it based on people's feedback and other factors that impact delivery and priorities. But we hope it gives the programme more clarity on where we're going and how we get there over the next 2-3 years!
```
Comment (claire-hughez) [inline app/posts/cross-programme/our-refreshed-vision.md:106]:
```suggestion
This is our vision and our strategy for getting there, but it's not static. We are continuously updating it based on people's feedback and other factors that impact delivery and priorities. But we hope it gives the programme more clarity on where we're going and how we get there over the next 2 to 3 years.
```

File: app/posts/cross-programme/our-refreshed-vision.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+Get in touch with Mili Malde, Deputy Service Owner for the CPD Programme, if you have any questions.
```
Comment (claire-hughez) [inline app/posts/cross-programme/our-refreshed-vision.md:108]: is it helpful listing her name with no contact detes?

**Review by claire-hughez — COMMENTED** [review]
looks good, some thoughts, some which defo arent a priority!

File: app/posts/cross-programme/our-refreshed-vision.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+## The kinds of changes we can expect to see differs between the ECF and NPQ reforms.
+
+A better understanding the types of changes gives us an element of control (even if not full control) as we can more effectively plan for it and bake it into our strategy.
+
+There are no current plans to introduce more NPQs to the [current suite](https://www.gov.uk/guidance/national-professional-qualification-npq-courses), but the department may add more NPQs in the future, albeit at a much slower rate. With the [Education Recovery Funding](https://www.gov.uk/government/publications/recovery-premium-funding/recovery-premium-funding) (additional support for schools during and the years following the Covid-19 pandemic) coming to an end, it is unlikely we\'ll continue to fund the same high number of NPQ courses and participants as we have done over the last few years. This funding could change year-on-year, as well as the eligibility criteria. However, the department will still want to monitor how many NPQs are being taken up so they can evaluate the impact of the programme. Our service will continue to play a key role in facilitating this.
```
[the remark this answers is not in the document]
Comment (peteryates) [reply app/posts/cross-programme/our-refreshed-vision.md:31]:
```suggestion
There are no current plans to introduce more NPQs to the [current suite](https://www.gov.uk/guidance/national-professional-qualification-npq-courses), but the department may add more NPQs in the future, albeit at a much slower rate. With the [Education Recovery Funding](https://www.gov.uk/government/publications/recovery-premium-funding/recovery-premium-funding) (additional support for schools during and the years following the Covid-19 pandemic) coming to an end, it is unlikely we'll continue to fund the same high number of NPQ courses and participants as we have done over the last few years. This funding could change year-on-year, as well as the eligibility criteria. However, the department will still want to monitor how many NPQs are being taken up so they can evaluate the impact of the programme. Our service will continue to play a key role in facilitating this.
```

**Pull request merged**